proto/remote_execution.proto

```diff
@@ -2173,6 +2173,13 @@ message IOStats {
   // The time taken to download the tree.
   int64 file_download_duration_usec = 3;
 
+  // Total number of inputs that were provisioned from the local cache rather
+  // than downloading from the remote cache.
+  int64 local_cache_hits = 7;
+
+  // Total time spent linking inputs from local cache, across all threads.
+  google.protobuf.Duration local_cache_duration = 8;
```

Comment:
The comment is unclear to me -- is this a wall time or sum(all thread times)?

I think we want walltime.

Reply:
Updated the comment to clarify - it's the sum of times across all threads.

I think wall time could be defined as something like `end_timestamp_of_last_hardlink - start_timestamp_of_first_hardlink` but I'm not sure how useful that would be, because today we have remote fetches interleaved with hardlinks. If there is at least 1 remote fetch in the action, then I think the wall time becomes hard to interpret?

The sum of all thread times is also kind of hard to interpret, but I think it more directly correlates to the amount of "work" that we had to do in order to set up the input root.

Reply:
The thing that's confusing about this is that there are other metrics in this proto, like `file_download_duration_usec` which are walltime measures.

The value of the walltime measure (I think?) is that it is directly measuring how long the user is waiting for something. It's also easy to interpret, where sum(all thread times) is hard to interpret especially because you don't have number of threads.